src/main/java/net/glowstone/block/blocktype/BlockRedstoneTorch.java

```diff
@@ -55,7 +55,7 @@ public void placeBlock(GlowPlayer player, GlowBlockState state, BlockFace face,
     @Override
     public void updatePhysics(GlowBlock me) {
```

Comment:
Redstone torches are faulty. Many standard redstone structures were working before, however, they are not working now. I am not sure if this is a result of this PR, but this PR should fix the issue, providing that we just start the pulse incorrectly.